draft-foudil-securitytxt.md

```diff
@@ -140,7 +143,7 @@ Encryption: https://example.com/pgp-key.txt
 ## Signature: {#signature}
 
 In order to ensure the authenticty of the security.txt file one SHOULD use the
-"Signature:" directive, which allows you to link to an external signature. External signature files should be
+"Signature:" directive, which allows you to link to an external signature. External signature files SHOULD be
 named "security.txt.sig" and also be placed under the /.well-known/ path.
 External signature files SHOULD be loaded over HTTPS.
```

Comment:
Do we want to add verbiage here about the possibility of an attacker subverting this, or leave that for the "Security Considerations" section?